pkg/converters/ingress/ingress.go

```diff
@@ -597,6 +611,27 @@ func (c *converter) addBackend(source *annotations.Source, hostname, uri, fullSv
 	return backend, nil
 }
 
+func (c *converter) syncBackendEndpointCookies(backend *hatypes.Backend) {
+	cookieAffinity := backend.CookieAffinity()
+	for _, ep := range backend.Endpoints {
+		if cookieAffinity {
+			switch backend.EpCookieStrategy {
+			default:
+				ep.CookieValue = ep.Name
+			case hatypes.EpCookiePodUid:
+				if ep.TargetRef != "" {
+					pod, err := c.cache.GetPod(ep.TargetRef)
```

Comment:
Update `--disable-pod-list` command-line doc that the pod list is also being used by session-cookie-preserve as well.

Reply:
Implemented in https://github.com/jcmoraisjr/haproxy-ingress/pull/667/commits/099b1f64b6c5a5fc3c48514f9d2e671b826acd71